easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,19 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are two easyconfig files for running GROMACS with AMD GPU acceleration (ROCm support via hipSYCL), that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus):
+
+    * MPI version for multi-node use, with [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) supporting offload to multiple GPUs 
+    ([GPU aware MPI](https://manual.gromacs.org/2023.2/install-guide/index.html#gpu-aware-mpi-support) enabled, support for PME decomposition)
+    * Both MPI and thread-MPI version with [VkFFT](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) recommended for a single GPU use (no support for a direct GPU communication, without PME decomposition)
```

Comment:
```suggestion
    * Both MPI and thread-MPI version with [VkFFT](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) recommended for a single GPU use (no support for direct GPU communication with thread-MPI, no support for GPU PME decomposition with VkFFT)
```

The above clarification is necessary because "direct GPU communication" aka GPU-aware communication is only supported with MPI (while in our CUDA backend it is also supported with thread-MPI).